django/contrib/auth/hashers.py

```diff
@@ -322,53 +323,76 @@ def encode(self, password, salt):
             parallelism=self.parallelism,
             hash_len=argon2.DEFAULT_HASH_LENGTH,
             type=argon2.low_level.Type.I,
+            version=self.version,
```

Comment:
Then we should drop this, according to https://github.com/hynek/argon2_cffi/blob/master/src/argon2/low_level.py#L58 it should already select the latest.